src/index.ts

```diff
@@ -0,0 +1,80 @@
+import './styles/style.css';
+import { createTemplate, IPersonData } from './template';
+import { createSession } from './xhr';
+
+interface IOptions {
+  host?: string;
+  apiKey: string;
+  parentId: string;
+  onSession: (err, response) => void;
+}
+
+interface IMountOptions {
+  formLabel?: any;
+  submitBtnText?: string;
+  loadingText?: string;
+}
+
+interface IParams {
+  person?: IPersonData;
+  vendorData?: string;
+}
+
+const Veriff = (options: IOptions) => {
+  const { host = 'https://api.veriff.me', apiKey, parentId, onSession } = options;
+  return {
+    params: {
+      person: {},
+    },
+    setParams(newParams: IParams) {
+      this.params = { ...this.params, ...newParams };
+    },
+    updateParams(newParams: IParams) {
+      this.params = { ...newParams };
+      this.form = createTemplate(parentId, {
+        person: this.params.person,
+        vendorData: this.params.vendorData,
+      });
+      this.form = this.assignSubmit(this.form);
+    },
+    form: this.form,
+    assignSubmit(form, loadingText = 'Loading...', submitBtnText = 'Start Verification') {
+      form.onsubmit = (e) => {
+        e.preventDefault();
+
+        const givenName = form.givenName ? form.givenName.value : this.params.person.givenName;
+        const lastName = form.lastName ? form.lastName.value : this.params.person.lastName;
+        const idNumber = form.idNumber ? form.idNumber.value : this.params.person.idNumber;
```

Comment:
are we keeping the id number in the JS SDK? talk to Maksim, maybe we can simply drop it. we'll keep existing integrations intact by versioning all new releases